corehq/apps/analytics/templates/analytics/analytics_js.html

```diff
@@ -12,4 +12,5 @@
   <script src="{% static 'analytix/js/hubspot.js' %}"></script>
   <script src="{% static 'analytix/js/drift.js' %}"></script>
   <script src="{% static 'analytix/js/appcues.js' %}"></script>
+  <script src="{% static 'hqwebapp/js/toggles.js' %}"></script>
```

Comment:
I'm sorry, I had thought that we'd already made feature flags available in web apps. I see you've figured it out, but it should be a little bit different - including `toggles.js` here means most pages will include it twice, because it's also in [base.html](https://github.com/dimagi/commcare-hq/blob/8a73ab58201c6a5f06b41e51b09dd4a759990eb7/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L422), and the toggle data is already available to all views because it's in a [context processor](https://github.com/dimagi/commcare-hq/blob/8a73ab58201c6a5f06b41e51b09dd4a759990eb7/corehq/util/context_processors.py#L128).

Can you work off of the branch in this PR to pull in the toggles infrastructure? https://github.com/dimagi/commcare-hq/pull/28813/

Reply:
Noting that PR is now merged, so you can just merge master.